xarray_sentinel/sentinel1.py

```diff
@@ -3,6 +3,7 @@
 import warnings
 
 import numpy as np
+import pandas as pd  # type: ignore
```

Comment:
Minor. I think dates / times can be manipulated with `numpy` alone, it would be nice to avoid importing `pandas` if possible (ATM only `pd.date_range` is really useful)